src/main/java/net/dv8tion/jda/api/entities/User.java

```diff
@@ -455,6 +465,70 @@ public String toString()
         }
     }
 
+    /**
+     * Represents the avatar decoration of a {@link User User}.
+     */
+    class AvatarDecoration
+    {
+
+        private final String decorationAvatarId;
+        private final String skuId;
+
+        public AvatarDecoration(String decorationAvatarId, String skuId)
+        {
+            this.decorationAvatarId = decorationAvatarId;
+            this.skuId = skuId;
+        }
+
+        /**
+         * The never-null SKU id of the {@link User User} decoration avatar.
+         *
+         * @return The never-null SKU id of the {@link User User} decoration avatar.
+         */
+        @Nonnull
+        public String getSkuId()
```

Comment:
How is this useful to a bot?

Reply:
I'm not sure how for now, but maybe JDA will add support for SKU later on? Should I just remove that entirely?